src/components/KeyValueEditor/KeyValueEditor.tsx

```diff
@@ -21,50 +33,88 @@ export interface TextInputProps extends DefaultProps {
  * - a dictionary (key value)
  * - or an array of items, then you need to pass the keyProp and valueProp
  * @param data
- * @param keyProp
- * @param valueProp
+ * @param keyLabel
+ * @param valueLabel
+ * @param keySeparator
  * @param readonly
  * @param onChange
  * @param className
+ * @param noDataMessage
+ * @param noDataForFilterMessage
  * @constructor
  */
 export const KeyValueEditor: FunctionComponent<TextInputProps> = ({
 	data,
-	keyProp,
-	valueProp,
+	keyLabel = 'Id',
+	valueLabel = 'Waarde',
+	keySeparator = '___',
 	readonly = false,
 	onChange = () => {},
 	className,
+	noDataMessage = 'Geen data',
+	noDataForFilterMessage = 'Geen data die voldoet aan de filter',
```

Comment:
Mij lijkt het dat we deze aan de client-side zullen moeten defaulten om translations toe te laten.

Reply:
Dit is een default backup, op de client kunnen we inderdaad vertaalbare props doorgeven. Maar moest je niets doorgeven zal hij die strings tonen.